### PR TITLE
Move code review slide to "how do we make it happen" section

### PR DESCRIPTION
**Pull request by standage**

### CODE REVIEW

File: short/index.html [hunk 1]
```diff
@@ -248,6 +236,18 @@
 </section>
 
 <section data-markdown>
+**Code Review**
+
+*   Academics review each other's papers; what about code?
+*   Re: code review, [Petre](http://arxiv.org/abs/1407.5648) found:
```
Comment (gvwilson) [inline short/index.html]: Remove "Re: code review, "

File: short/index.html [hunk 2]
```diff
@@ -248,6 +236,18 @@
...
+    *   Requires domain knowledge to be useful
+    *   No point doing it at submission time
+*   But domain experts are scarce
+*   No incentives to review someone else's thesis project code
+*   At present, only sustainable within team projects
```
Comment (gvwilson) [inline short/index.html:247]: This wasn't a finding, but a conclusion.

File: short/index.html (hunk 1 above)
Comment (standage) [reply short/index.html]: Done.

File: short/index.html (hunk 2 above)
Comment (standage) [reply short/index.html:247]: Are you referring to the last bullet? The "findings" should refer only to the two sub-bullets, no?

Comment (gvwilson) [reply short/index.html:247]: Yeah, I'm nit-picking - ignore me.

**Pull request merged**